tests/impls/rank_relevancy/test_random_sampling.py

```diff
@@ -0,0 +1,68 @@
+import numpy
+from typing import Dict, List, Sequence, Iterable
+
+from smqtk_relevancy.interfaces.rank_relevancy import RankRelevancy
+from smqtk_relevancy.impls.rank_relevancy.random_sampling import (
+    RankRelevancyWithRandomFeedback,
+)
+from smqtk_descriptors import DescriptorElement
+
+
+def test_is_usable() -> None:
+    assert RankRelevancyWithRandomFeedback.is_usable()
+
+
+class DummyRankRelevancy(RankRelevancy):
+    def get_config(self) -> Dict:
+        return {}
```

Comment:
If this is never called, we can collapse the likely untested line here by writing (e.g. codecov would complain about this)
```
def get_config(self) -> Dict: ...
```

Reply:
Done.